src/components/SelectionCpuLevel.tsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from "react"
+type Level = "強い" | "普通" | "弱い"
+
+export const SelectionCpuLevel: React.FC = () => {
+  const [level, setLevel] = useState<Level>("強い")
+
+  const handleLevelChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    const value = e.target.value as Level
+    setLevel(value)
```

Comment:
ここ、俺が言っといてなんだけどこう書いたほうがスマートそう

```suggestion
    setLevel(e.target.value as Level)
```